### PR TITLE
Convert chrony configuration instructions to Butane

### PR DESCRIPTION
**Pull request by bgilbert**
Avoid separately applying base64 to input files.

cc @bobfuru @miabbott

### CODE REVIEW

File: modules/ipi-install-configuring-ntp-for-disconnected-clusters.adoc [hunk 1]
```diff
@@ -23,216 +23,152 @@ image::152_OpenShift_Config_NTP_0421.svg[Configuring NTP for disconnected cluste
...
-  config:
-    ignition:
-      version: 2.2.0
-    storage:
-      files:
-        - contents:
-            source: data:text/plain;charset=utf-8;base64,BASE64ENCODEDCONFIGFILE<1>
-          filesystem: root
-          mode: 0644
-          path: /etc/control-plane-chrony.conf
```
Comment (bgilbert) [inline modules/ipi-install-configuring-ntp-for-disconnected-clusters.adoc:112]: This appears to be an error, so I've changed it to `/etc/chrony.conf`.

File: modules/ipi-install-configuring-ntp-for-disconnected-clusters.adoc [hunk 2]
```diff
@@ -23,216 +23,152 @@ image::152_OpenShift_Config_NTP_0421.svg[Configuring NTP for disconnected cluste
...
-  config:
-    ignition:
-      version: 2.2.0
-    storage:
-      files:
-        - contents:
-            source: data:text/plain;charset=utf-8;base64,BASE64ENCODEDCONFIGFILE<1>
-          filesystem: root
-          mode: 0644
-          path: /etc/worker-chrony.conf
```
Comment (bgilbert) [inline modules/ipi-install-configuring-ntp-for-disconnected-clusters.adoc:169]: Likewise.

File: modules/ipi-install-configuring-ntp-for-disconnected-clusters.adoc [hunk 3]
```diff
@@ -23,216 +23,154 @@ image::152_OpenShift_Config_NTP_0421.svg[Configuring NTP for disconnected cluste
...
-[source,terminal]
-.Configuration file example
+[source,yaml]
+.Butane config example
 ----
-# This example MachineConfig replaces ~/worker-chrony.conf
-apiVersion: machineconfiguration.openshift.io/v1
-kind: MachineConfig
+variant: openshift
+version: 4.8.0
```
Comment (bobfuru) [inline modules/ipi-install-configuring-ntp-for-disconnected-clusters.adoc:95]: For my future knowledge, will the version in these examples need to be updated with each OCP minor release?

**Review by bobfuru — COMMENTED** [review]
One minor nit and one question and otherwise LGTM. 🙌

**Comment by bobfuru** [comment]
FYI @johnwilkins, some changes from what was added in https://github.com/openshift/openshift-docs/pull/31853

**Comment by bgilbert** [comment]
Updated.

File: modules/ipi-install-configuring-ntp-for-disconnected-clusters.adoc (hunk 3 above)
Comment (bgilbert) [reply modules/ipi-install-configuring-ntp-for-disconnected-clusters.adoc:95]: It's not necessary.  Old configs should be forward-compatible.

**Review by miabbott — APPROVED** [review]
LGTM

**Pull request merged**